scripts/README.md

```diff
@@ -0,0 +1,94 @@
+# Updating tormays -material to environemnts
+
+General overview:
+
+These instructions will guide through various steps. Actual steps
+taken is dependent on the task at hand.
+
+- upload generated polygons to Haitaton file share (for others to access)
+- download polygons from file share to local disk (if needed)
+- update GIS materials from local disk to local dev database
+- update to dev
+- update to test
+
+Support for staging and production environments can be easily added later on, if needed.
+
+Prerequisites:
+
+- tormays polygon gis material is already generated
+
+## Prerequisite: Generate tormays polygons
+
+Generating tormays-polygon data is instructed in `gis-material-update/README.md`
+
+## Upload generated polygons to Haitaton file share
+
+Script run will resolve fileshare name and detect processed materials from
+processing output directory (`gis-material-update/haitaton-gis-output`).
+
+Commands are printed to stdout during script run, if processed material is found from directory.
+
+Following steps will provide commands to run:
+
+- ensure VPN is active and you are logged to OpenShift
+- select correct project, as instructed in oc login `oc project <projectname>`
+- run `sh hel-gis-print-upload-commands.sh`
+- copy and paste selected commands to run
+
+## Download from file share to local disk
+
+Following command will download files from file share.
+
+Output directory and tormays polygon files are configured in separate configuration file.
+
+Following steps will download files to local directory:
+
+- ensure VPN is active
+- run `sh hel-gis-data-download.sh <dev|test>`
+
+## Update material from local disk to local development database
+
+Local directory and tormays polygon files are configured in separate configuration file.
+
+- ensure VPN is passive (VPN blocks local port usage for some deployments)
+- run `sh hel-gis-data-import-to-db.sh <directory>`
```

Comment:
It should be specified what directory this refers to.

Reply:
Kun koitan laittaa äsken generoimiani materiaaleja paikalliseen kantaan, joku menee pieleen:
```
❯ sh hel-gis-data-import-to-db.sh gis-material-update/haitaton-gis-output
[1/      12] Importing 'tormays_buses_polys.gpkg'
FAILURE:
Unable to open datasource `gis-material-update/haitaton-gis-output/tormays_buses_polys.gpkg' with the following drivers.
  -> `PCIDSK'
  -> `PDS4'
  -> `VICAR'
  -> `JP2OpenJPEG'
  -> `PDF'
  -> `MBTiles'
  -> `EEDA'
  -> `OGCAPI'
  -> `ESRI Shapefile'
  -> `MapInfo File'
  -> `UK .NTF'
  -> `LVBAG'
  -> `OGR_SDTS'
  -> `S57'
  -> `DGN'
  -> `OGR_VRT'
  -> `Memory'
  -> `CSV'
  -> `GML'
  -> `GPX'
  -> `KML'
  -> `GeoJSON'
  -> `GeoJSONSeq'
  -> `ESRIJSON'
  -> `TopoJSON'
  -> `OGR_GMT'
  -> `GPKG'
  -> `SQLite'
  -> `WAsP'
  -> `PostgreSQL'
  -> `OpenFileGDB'
  -> `DXF'
  -> `CAD'
  -> `FlatGeobuf'
  -> `Geoconcept'
  -> `GeoRSS'
  -> `VFK'
  -> `PGDUMP'
  -> `OSM'
  -> `GPSBabel'
  -> `OGR_PDS'
  -> `WFS'
  -> `OAPIF'
  -> `EDIGEO'
  -> `SVG'
  -> `Idrisi'
  -> `ODS'
  -> `XLSX'
  -> `Elasticsearch'
  -> `Carto'
  -> `AmigoCloud'
  -> `SXF'
  -> `Selafin'
  -> `JML'
  -> `PLSCENES'
  -> `CSW'
  -> `VDV'
  -> `MVT'
  -> `NGW'
  -> `MapML'
  -> `GTFS'
  -> `TIGER'
  -> `AVCBin'
  -> `AVCE00'
  -> `HTTP'
```

Näyttää kuitenkin olevan olemassa:
```
❯ ls -al gis-material-update/haitaton-gis-output/tormays_buses_polys.gpkg
-rw-r--r--  1 topias.heinonen  staff  11223040 Feb  3 16:54 gis-material-update/haitaton-gis-output/tormays_buses_polys.gpkg
```

Reply:
Sama muillakin generoiduilla tiedostoilla:
```
❯ sh hel-gis-data-import-to-db.sh gis-material-update/haitaton-gis-output
[1/      12] Importing 'tormays_buses_polys.gpkg'
FAILURE:
Unable to open datasource `gis-material-update/haitaton-gis-output/tormays_buses_polys.gpkg' with the following drivers.
...
```

Filesharesta haetut näyttää toimivan:
```
❯ sh hel-gis-data-import-to-db.sh tormays_polys
[1/      12] Importing 'tormays_buses_polys.gpkg'
Imported 'tormays_buses_polys.gpkg'
[2/      12] Importing 'tormays_central_business_area_polys.gpkg'
Imported 'tormays_central_business_area_polys.gpkg'
[3/      12] Importing 'tormays_critical_area_polys.gpkg'
Imported 'tormays_critical_area_polys.gpkg'
[4/      12] Importing 'tormays_cycleways_basic_polys.gpkg'
Imported 'tormays_cycleways_basic_polys.gpkg'
[5/      12] Importing 'tormays_cycleways_main_polys.gpkg'
Imported 'tormays_cycleways_main_polys.gpkg'
[6/      12] Importing 'tormays_cycleways_priority_polys.gpkg'
Imported 'tormays_cycleways_priority_polys.gpkg'
[7/      12] Importing 'tormays_street_classes_polys.gpkg'
Imported 'tormays_street_classes_polys.gpkg'
[8/      12] Importing 'tormays_trams_polys.gpkg'
Imported 'tormays_trams_polys.gpkg'
[9/      12] Importing 'tormays_volumes15_polys.gpkg'
Imported 'tormays_volumes15_polys.gpkg'
[10/      12] Importing 'tormays_volumes30_polys.gpkg'
Imported 'tormays_volumes30_polys.gpkg'
[11/      12] Importing 'tormays_ylre_classes_polys.gpkg'
Imported 'tormays_ylre_classes_polys.gpkg'
[12/      12] Importing 'tormays_ylre_parts_polys.gpkg'
Imported 'tormays_ylre_parts_polys.gpkg'
Import took 2.18 minutes
```

Reply:
Näköjään toi itseluodutkin toimivat, ken ne ensin kopioi tonne `tormays_polys`-hakemistoon.

Reply:
Kyllä, tässä oli toiminta jäänyt puolitiehen. Siirryin toteutuksessa takaisin käyttämään kiinteää `tormays_polys` -hakemistoa, sillä se on hivenen vikasietoisempi kuin käyttäjän antama parametri.